package.json

```diff
@@ -27,38 +27,38 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "chalk": "^4.1.1",
-    "cli-truncate": "^2.1.0",
-    "commander": "^7.2.0",
-    "cosmiconfig": "^7.0.0",
-    "debug": "^4.3.1",
+    "cli-truncate": "2.1.0",
+    "colorette": "^1.4.0",
+    "commander": "^8.2.0",
```

Comment:
This dependency upgrade bumps the minimum node.js version from 10 to 12. So maybe that should be considered as breaking change instead?

Reply:
It for sure should have been! Thanks for pointing this out. cc @iiroj

Reply:
Oops! Maybe a `fix` to downgrade? Or should we bump our version now that 16 is in LTS and 10 is getting old?

Reply:
I'd bump and release as a major TBH

Reply:
I figured we could go all the way and convert _lint-staged_ to ESM: https://github.com/okonet/lint-staged/pull/1038

Reply:
Actually, we already require Node.js 12.13, we just don't enforce it via the `engines.node` key:

https://github.com/okonet/lint-staged/blob/e035b80e39da355da57c02db6565b55271ab1afa/bin/lint-staged.js#L20